assets/js/src/settings/pages/signup_confirmation/email_content.tsx

```diff
@@ -1,4 +1,6 @@
-import React from 'react';
+import React, { Fragment } from 'react';
+import ReactStringReplace from 'react-string-replace';
```

Comment:
I don't see this being used, am I missing something?

Reply:
You are right, I will remove it.

Reply:
I wonder why didn't eslint complain. I thought we have it configured like that.